packages/bpk-component-button/src/BpkButtonBase.module.scss

```diff
@@ -45,6 +45,6 @@
   }
 
   svg {
-    fill: currentColor;
+    fill: currentcolor;
```

Comment:
this too

Reply:
Yep

![Screenshot 2022-09-05 at 16 21 31](https://user-images.githubusercontent.com/8831547/188480654-de1223b8-c300-4a18-9cc5-5d5b394cc14d.png)